repo/packages/N/neo4j-proxy/0/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "object",
+  "properties": {
+    "neo4j-proxy": {
+      "description": "Neo4j specific configuration properties",
+      "type": "object",
+      "properties": {
+        "auth-username": {
+          "description": "Username for authentication",
+          "type": "string",
+          "default": "neo4j"
+        },
+        "auth-password": {
+          "description": "Password for authentication",
+          "type": "string",
+          "default": "dcos"
```

Comment:
This configuration. This point is very confusing, because it is semantically like `If you don`t have configured authentication information, this is your login`. @jexp of neo4j opened an ticket to change this in something like my quote.